feature-utils/poly-look/src/react-components/banners/notificationBanner.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import "./notificationBanner.css";
+
+import standard from "../../static/images/icons/xmark-standard.svg";
+import success from "../../static/images/icons/xmark-success.svg";
+import error from "../../static/images/icons/xmark-error.svg";
+import warning from "../../static/images/icons/xmark-warning.svg";
+
+export const types = {
```

Comment:
Maybe "kinds" and not "types" (since this means something in a programming context), but not really important.

Reply:
I renamed it to `notificationTypes` so it's not that generic.